lib/chef/knife/azurerm_base.rb

```diff
@@ -27,6 +27,10 @@ class Chef
   class Knife
     module AzurermBase
 
+      ## azure-xplat-cli versio that introduced deprecation of Windows Credentials
+      ## Manager (WCM) usage for authentication credentials storage purpose ##
+      XPLAT_VERSION_WITH_WCM_DEPRECATED ||= "0.10.5"
```

Comment:
I'm not that worried, either way. That shouldn't happen though. Perhaps it's related to the user of the pattern `require File.expand_path('../azurerm_base', __FILE__)` in the project.